django/cantusdb_project/main_app/templates/feast_list.html

```diff
@@ -0,0 +1,113 @@
+{% extends "base.html" %}
+{% load helper_tags %}
+{% block content %}
+<script>
+    // Make sure the select components keep their values across multiple GET requests
+    // so the user can "drill down" on what they want
+    window.onload = function () {
+        $("#month-filter").val("{{ request.GET.month }}")
+        $("#order-by").val("{{ request.GET.ordering }}")
+        $("#date-filter").val("{{ request.GET.date }}")
+    }
+
+</script>
+<div class="mr-3 p-3 col-md-10 mx-auto bg-white rounded">
+    <h3>List of Feasts</h3>
+    <small>Displaying {{page_obj.start_index}}-{{page_obj.end_index}} of <b>{{page_obj.paginator.count}}</b></small>
+    <form method="get">
+        <div class="form-row align-items-end">
+            <div class="form-group m-1 col-lg-5">
+                <label for="feast-search">Search feasts/feast codes</label>
+                <input type="text" class="form-control form-control-sm"
+                    placeholder="Type any part of a word or feast code" name="q" value="{{ request.GET.q}}"
+                    id="feast-search">
+            </div>
+            <div class="form-group m-1 col-lg">
+                <label for="date-filter">Temp/Sanc</label>
+                <select id="date-filter" name="date" class="form-control form-control-sm">
+                    <option value="">- Any -</option>
+                    <option value="1">Temporale</option>
+                    <option value="2">Sanctorale</option>
+                </select>
+            </div>
+            <div class="form-group m-1 col-lg">
+                <label for="order-by">Sort By</label>
+                <select class="form-control form-control-sm" id="order-by" name="ordering">
+                    <option value="name">Feast Name</option>
+                    <option value="feast_code">Feast Code</option>
+                </select>
+            </div>
+            <div class="form-group m-1 col-lg">
+                <label for="month-filter">Month</label>
+                <select name="month" class="form-control form-control-sm" id="month-filter">
+                    <option value="">- Any -</option>
+                    <option value="11">November</option>
+                    <option value="12">December</option>
+                    <option value="1">Jan</option>
+                    <option value="2">Feb</option>
+                    <option value="3">Mar</option>
+                    <option value="4">Apr</option>
+                    <option value="5">May</option>
+                    <option value="6">June</option>
+                    <option value="7">July</option>
+                    <option value="8">August</option>
+                    <option value="9">September</option>
+                    <option value="10">October</option>
```

Comment:
Is there a reason for the order and why some months are abbreviated while others aren't?

Reply:
Good catch